packages/utils/odsp-doclib-utils/src/odspRequest.ts

```diff
@@ -13,7 +14,10 @@ export async function getAsync(
 	url: string,
 	authRequestInfo: IOdspAuthRequestInfo,
 ): Promise<Response> {
-	return authRequest(authRequestInfo, async (config: RequestInit) => fetch(url, config));
+	return authRequest(
+		authRequestInfo,
+		async (config: RequestInit) => fetch(url, config) as unknown as Response,
```

Comment:
ouch - what is going on here?

Reply:
Simplified to just 'as Response' (ditto elsewhere).